lib/Text/Amuse/Output.pm

```diff
@@ -1353,12 +1353,20 @@ sub format_single_link {
             return "\\hyperref{}{amuse}{$linkname}{$linkname}";
         }
     }
+
+    my $url = $self->_url_safe_escape($link);
+    my $desc = $self->manage_regular($link);
```

Comment:
Here it should use ->safe($link) instead of ->manage_regular to avoid interpolation.